runtime/doc/vim_diff.txt

```diff
@@ -49,27 +49,27 @@ are always available and may be used simultaneously in separate plugins.  The
 
 See |nvim-intro| for a list of Nvim's largest new features.
 
-Meta key chords are recognized (even in the terminal).
-- <M-1>, <M-2>, ...
-- <M-BS>, <M-Del>, <M-Ins>, ...
-- <M-/>, <M-\>, ...
-- <M-Space>, <M-Enter>, <M-=>, <M-->, <M-?>, <M-$>, ...
+Meta-key chords are recognized (even in the terminal).
+  <M-1>, <M-2>, ...
+  <M-BS>, <M-Del>, <M-Ins>, ...
+  <M-/>, <M-\>, ...
+  <M-Space>, <M-Enter>, <M-=>, <M-->, <M-?>, <M-$>, ...
 
-CTRL-SHIFT-... key chords are distinguished from CTRL-... variants (even in
+`CTRL-SHIFT-...` key chords are distinguished from `CTRL-...` variants (even in
 the terminal).
-- <C-Tab>, <C-S-Tab>
-- <C-BS>, <C-S-BS>
-- <C-Enter>, <C-S-Enter>
+  <C-Tab>, <C-S-Tab>
+  <C-BS>, <C-S-BS>
+  <C-Enter>, <C-S-Enter>
 
 Events:
```

Comment:
I thought I wouldn't like a lack of dashes, but I think it looks cleaner so :+1:

One nit though: IMO these headings should either have the colon removed to match what's done in section 5, or the headings in section 5 should have colons added.